lib/index.js

```diff
@@ -7,39 +7,36 @@ const path = require("path");
  * First function to be called after running a flag. This is a check,
  * to match the flag with the respective require.
  *
- * @param {String} command - which feature to use
- * @param {Object} args - arguments from the CLI
- * @returns {Function} invokes the module with the supplied command
+ * @param	{String}	command - which feature to use
+ * @param	{Object}	args - arguments from the CLI
+ * @returns	{Function}	invokes the module with the supplied command
  *
  */
 
 module.exports = function initialize(command, args) {
 	const popArgs = args ? args.slice(2).pop() : null;
+	const cmdArgs = args.slice(2).length === 1 ? [] : [popArgs];
+	// default config filename
+	const configFile = cmdArgs.length ? cmdArgs[0] : "webpack.config.js";
```

Comment:
`"webpack.config.js"` should be inside a constant
```js
const DEFAULT_WEBPACK_CONFIG_FILENAME = "webpack.config.js";
```
A bit long but it gives clear idea lol

Reply:
Added the variable.